java/src/main/java/org/rocksdb/Checkpoint.java

```diff
@@ -5,6 +5,8 @@
 
 package org.rocksdb;
 
+import java.util.*;
```

Comment:
CI complains about this import: "Unused import 'java.util.*'"

Reply:
Thanks,  fix it